Dockerfile

```diff
@@ -5,14 +5,21 @@ COPY signer.py /
 COPY requirements.txt /
```

Comment:
What about putting the COPY steps after the install steps? So that we don't reinstall everything every time a file changes.

```
COPY requirements.txt /
RUN pip3 install -r /requirements.txt

COPY src/. /src/
RUN chmod 755 /src/start-remote-signer.sh

COPY signer.py /

ENTRYPOINT ["/src/start-remote-signer.sh"]
```

Reply:
Hmmm, yes, I thought of that and then forgot about it. I'll do it now.